app-editors/qhexedit2/metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>coyote@bks.tv</email>
+        <name>Victor Kustov</name>
+        <description>Proxied-maintainer. Assign bugs to him</description>
+    </maintainer>
+    <maintainer type="person">
+        <email>zlogene@gentoo.org</email>
+        <name>Mikle Kolyada</name>
+    </maintainer>
+    <maintainer type="person">
+        <email>maksbotan@gentoo.org</email>
+        <name>Maxim Koltsov</name>
+    </maintainer>
+    <maintainer type="project">
+        <email>proxy-maint@gentoo.org</email>
+        <name>Proxy Maintainers</name>
+    </maintainer>
+    <use>
+        <flag name="doc">Install documentation</flag>
+        <flag name="example">Build and install example - Qt5 hex editor</flag>
+        <flag name="widget">Build with PyQt5 widget and install it</flag>
```

Comment:
widget could use a better definition. What's the benefit? We know it's going to build/install something. What is it?

The might be better removed and use the python use flag instead since I'm guessing it controls whether Python bindings are installed.

Reply:
- remove "doc" "widget" from metadata, 
- replace metadata "example" by "gui"
- change USE flags: "widget" -> "python", "example" -> "gui"
Is it correct?